app/helpers/appointment_helper.rb

```diff
@@ -0,0 +1,13 @@
+module AppointmentHelper
+
+  def doctor_all
+    Doctor.all.collect { |p| [ p.user_name, p.id ] }
```

Comment:
- đặt  tên action là doctors
- 
```suggestion
    Doctor.collect { |doctor| [ doctor.user_name, doctor.id ] }
```

Reply:
anh ơi em bỏ all đi nó lại báo lỗi không xác định method collect  ạ